.github/workflows/lint-and-test.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           fi
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.2.0
+        uses: helm/kind-action@v1.4.0
```

Comment:
There is no breaking changes. :eyes:
https://github.com/helm/kind-action/releases/tag/v1.4.0